lib/tasks/user.rake

```diff
@@ -11,6 +11,19 @@ namespace :user do
     end
   end
 
+  desc "Backfill UX testing emails comms field in batches"
+  task backfill_ux_testing_email_field: :environment do
+    User.find_in_batches do |users|
```

Comment:
Should this have a `.select(:id)` to lighten the amount of data needed?

Reply:
Good shout - i'll look to add that in and get this out.